src/layer/poi.js

```diff
@@ -3,50 +3,76 @@ import * as Color from "../constants/color.js";
 
 var iconDefs = {
   bar: {
-    subclasses: ["bar", "beer", "pub"],
+    classes: {
+      bar: ["bar"],
+      beer: ["beer", "pub"],
+    },
     sprite: "poi_martini_glass",
     color: Color.poi.consumer,
     description: "Bar or pub",
   },
   coffee: {
-    subclasses: ["cafe"],
+    classes: {
+      cafe: ["cafe"],
+    },
     sprite: "poi_coffee_cup",
     color: Color.poi.consumer,
     description: "Coffee shop",
   },
   hospital: {
-    subclasses: ["hospital"],
+    classes: {
+      hospital: ["hospital"],
+    },
     sprite: "poi_hospital",
     color: Color.poi.infrastructure,
     description: "Hospital",
   },
   medical: {
-    subclasses: ["doctors", "clinic"],
+    classes: {
+      hospital: ["doctors", "clinic"],
```

Comment:
```suggestion
      hospital: ["clinic"],
      doctors: ["doctors"],
```
I think this suggestion better matches the OpenMapTiles Schema